### ISSUE TITLE
WCAG - Focus lands in wrong place upon form submission when error present

### ISSUE THREAD

**Issue by fionamorrison23** [civictheme/uikit#519]
### Summary

When a user submits a webform, when there is an error with the submission, focus moves to the top of the page (before the header). The form has an error summary at the top of the page and focus should be set there.

### Steps to reproduce

1. Open webform, fill in fields but leave one required field empty
2. Submit the form
3. Note where the focus lands

### Observed outcome

Focus lands at the top of the page on the skip link

### Expected outcome

Focus should land on the error message box at the top of the form
